composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "symfony/phpunit-bridge": "^2.7",
-        "phpunit/phpunit": "~5.1",
+        "phpunit/phpunit": "5.2",
```

Comment:
you should set a new `require-dev` for php version (such as `"php": ">=5.6"`) then, because: 
> PHPUnit 5.2 is supported on PHP 5.6 and PHP 7